maestro-client/src/main/resources/maestro-driver-ios-config.xctestrun

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
long-term should those files be under the source control? for me they can be stored elsewhere, i.e. gcloud

Reply:
I think its fine to keep it here since these are intentionally packaged in jar.